modules/configuring-pods-secondary-network.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/configuring-secondary-networks.adoc
+
+:_content-type: CONCEPT
+[id="configuring-pods-secondary-network_{context}"]
+= Configuring pods for secondary networks
+
+You must specify the secondary network attachments through the `k8s.v1.cni.cncf.io/networks` annotation.
+
+The following example provisions a pod with two secondary attachments, one for each of the attachment configurations presented in this guide.
```

Comment:
maybe we want to also show an example where the static IP for a pod is configured.

You can refer to this (currently unmerged) PR: https://github.com/ovn-org/ovn-kubernetes/pull/3454/files

Reply:
I can add this tomorrow.

Reply:
Added